editor/project_manager.cpp

```diff
@@ -2490,10 +2490,9 @@ ProjectManager::ProjectManager() {
 	project_order_filter = memnew(ProjectListFilter);
 	project_order_filter->add_filter_option();
 	project_order_filter->_setup_filters(sort_filter_titles);
-	project_order_filter->set_filter_size(150);
+	project_order_filter->set_filter_size(180);
```

Comment:
Does this size already take `EDSCALE` into account? It'd be good to check whether this behaves correctly when the editor scale is set to say, 150% or 200%.

Reply:
Yes, `set_filter_size()` applies `EDSCALE`:

https://github.com/godotengine/godot/blob/03e84abfa5a70fd2ea8b2df3ffea0a6a6c8c19e1/editor/project_manager.cpp#L2863-L2865